### ISSUE TITLE
A new version of the transaction signature algorithm.

### ISSUE THREAD

**Issue by xianaofei** [zcash/zcash#3398]
With regard to the latest version of zcash (such as 1.1.1), it is hoped that we can help to build the signature algorithm step after the transaction.
HashPrevouts, hashSequence and so on

**Comment by xianaofei** [comment]
hashPrevouts：
  BLAKE2b-256（'ZcashPrevoutHash'，b''）
= d53a633bbecf82fe9e9484d8a0e727c73bb9e68c96e72dec30144f6a84afa136
I want to know that this algorithm implements BLAKE2b-256

**Comment by ioptio** [comment]
Yes it's BLAKE2b-256 as specified in https://github.com/zcash/zips/blob/master/zip-0143.rst

Are you looking for anything else?
